_i18n/en.yml

```diff
@@ -599,24 +605,46 @@ policies:
     or [contact us](site.baseurl/contact).
 help_pages:
   changing-settings:
-    how-do-i-change-my-email-address: To change your email address, select Edit next
-      to the email address. Enter the new email address and submit it. Then check
-      your email account and look for an email from login.gov. Open the email and
-      click on the link provided. Clicking the link will redirect you back to login.gov.
-      Once you reach login.gov your new email address is established.
+    how-do-i-change-my-email-address: To change the email associated with your login.gov account, go to [https://www.login.gov/](https://www.login.gov/) and click “Manage Account” to sign into your login.gov account. Select “edit” next to the email address. Enter the new email address and submit it. Then check your email account and look for an email from login.gov. Open the email and click on the link provided. Clicking the link will redirect you back to login.gov. Once you reach login.gov, your new email address is established.
     how-do-i-change-my-password: |-
-      To change your password, sign in at login.gov and go to your Profile page. If you want to change your password, select Edit next to password, enter the new password and submit your change. Passwords must be at least 8 characters but otherwise there are no restrictions. You can even use more than one word with spaces to get to 8 characters.
+      To change your password, go to https://www.login.gov/ and click “Manage Account” to sign into your login.gov account. Select “Edit” next to password, enter the new password and submit your change. Passwords must be at least 9 characters but otherwise there are no restrictions. You can even use more than one word with spaces to get to 9 characters. Instead, try making a long and complex password by using a phrase or a series of words that only you recognize. And please make it different than any other passwords you use to sign into other accounts such as your bank account or email. Using the same passwords for your email, financial and social media accounts makes identity theft easier.
```

Comment:
In general, we shouldn't be asking folks to click things. Select, choose, follow are all good verbs. Click implies a mouse which is not a safe assumption.

Reply:
@clantosswett ☝️

Reply:
Let's go with select